src/components/apis/list-of-apis/react/runtime/ApiListRuntime.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from "react";
+import { useEffect, useState } from "react";
+import { FluentProvider, Spinner } from "@fluentui/react-components";
+import { Resolve } from "@paperbits/react/decorators";
+import { SearchQuery } from "../../../../../contracts/searchQuery";
+import * as Constants from "../../../../../constants";
+import { Api } from "../../../../../models/api";
+import { ApiService } from "../../../../../services/apiService";
+import { RouteHelper } from "../../../../../routing/routeHelper";
+import { Pagination } from "../../../../utils/react/Pagination";
+import { TableListInfo, TLayout } from "../../../../utils/react/TableListInfo";
+import { fuiTheme } from "../../../../../constants/fuiTheme";
+import { ApisTable } from "./ApisTable";
+import { ApisCards } from "./ApisCards";
+import { TApisData } from "./utils";
+
+export interface ApiListProps {
+  allowSelection?: boolean;
+  showApiType?: boolean;
+  defaultGroupByTagToEnabled?: boolean;
+  detailsPageUrl: string;
+
+  layoutDefault: TLayout | undefined; // TODO remove undefined once finished
+}
+
+const loadData = async (apiService: ApiService, query: SearchQuery, groupByTags?: boolean) => {
```

Comment:
in general, in React it's better to have inside the function as little as possible, because on every rerender it would be recreating the whole function

Reply:
Ok. Makes sense. Still I think it should be loadApi, how do you think?